hs_access_control/management/commands/groups_with_public_resources.py

```diff
@@ -0,0 +1,40 @@
+"""
+This prints a list of publicly accessible resources in a group
```

Comment:
This management command doesn't seem very useful as it's just a wrapper around GroupAccess that prints to the console.  Also the functions defined here aren't used.  delete?

Reply:
This is a useful debugging step in Communities Management. As a larger topic, I'd like to discuss this with you and Alva when he returns.

Reply:
The two methods still aren't used at all and should be removed.  The information may be useful, but I don't think it belongs in a management command.  Someone could get this information just as quickly by logging into the shell and calling `GroupAccess.groups_with_public_resources()`.